GUI/Main.resx

```diff
@@ -13774,4 +13774,95 @@
         AAD//wAA//8AAP//AAD//wAA//8AAA==
 </value>
   </data>
-</root>
\ No newline at end of file
+  <data name="$this.Localizable" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="$this.Language" type="System.Globalization.CultureInfo, mscorlib">
+    <value>(Default)</value>
```

Comment:
What does this do? It's just in this .resx, not in others.
Should I change `(Default)` to something different in the `Main.de-DE.resx`, or discard it, or leave it?

Reply:
All of the tutorials/answers that I found online said to set that property. I think it controls which language VS populates when you edit properties?

Reply:
So, just let it as is in `Main.de-DE.resx`?

Reply:
I think this property doesn't need to be included in `.de-DE.resx`.